CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# dbt_recharge version.version
+# dbt_recharge v0.3.1
+[PR #27](https://github.com/fivetran/dbt_recharge/pull/27) includes the following changes:
+
+- For Quickstart users, removed `recharge__line_item_enhanced` from the public models list in `quickstart.yml` since the model is disabled by default.
+- In `recharge__charge_line_item_history`, updated the discounts section to pull directly from the `charges` table. This is because we are currently unable to determine discounts on a charge_line level. Therefore, we have decided to maintain discounts at the charge level. In a future release, we may re-examine calculating discounts on the charge_line level when we have more complete data.
```

Comment:
How significant of a change is this? Will the data drastically change for customers? I want to be mindful of not introducing a major metric update in a patch fix. Let me know your thoughts.